README.md

```diff
@@ -1 +1,45 @@
-# newman-reporter-html
\ No newline at end of file
+# newman-reporter-html
+Advanced HTML reporter for [newman](https://github.com/postmanlabs/newman).
+
+## Install
+> The installation should be global if newman is installed globally, local otherwise. (Replace -g from the command below with -S for a local installation)
+
+```terminal
+$ npm install -g newman-reporter-html
+```
+
+## Usage
+
+```terminal
+$ newman run https://www.getpostman.com/collections/631643-f695cab7-6878-eb55-7943-ad88e1ccfd65-JsLv -r html
+```
+
+### Options
+
+| CLI Option  | Description       |
+|-------------|-------------------|
+| `--reporter-html-export <path>` | Specify a path where the output HTML file will be written to disk. If not specified, the file will be written to `newman/` in the current working directory. |
+| `--reporter-html-template <path>` | Specify a path to the custom template which will be used to render the HTML report. This option depends on `--reporter html` and `--reporter-html-export` being present in the run command. If this option is not specified, the [default template](https://github.com/postmanlabs/newman-reporter-html/blob/develop/lib/template-default.hbs) is used |
+
+Custom templates (currently handlebars only) can be passed to the HTML reporter via `--reporter-html-template <path>` with `--reporters html` and `--reporter-html-export`.
+The [default template](https://github.com/postmanlabs/newman-reporter-html/blob/develop/lib/template-default.hbs) is used in all other cases.
+
```

Comment:
Need to mention the programmatic options Newman will start accepting once this reporter is installed. 

There has to be a troubleshooting section
- when reporter is not found